themes/cucumber-sb/static/js/site.js

```diff
@@ -118,6 +118,13 @@ function getLangFromUrl(){
     return lang
 }
 
+function showDefaultLang(){
+  if(!supportedLanguages.includes(localStorage.getItem('language'))){
+    showOnly(defaultLanguage)
+  }else{
+    showOnly(localStorage.getItem('language'))
+  }
```

Comment:
```suggestion
  if(supportedLanguages.includes(localStorage.getItem('language'))){
    showOnly(localStorage.getItem('language'))
  }else{
    showOnly(defaultLanguage)
  }
```

A very minor point: We could flip the if so that the current else case "show only the localStorage-stored language" becomes the primary.

One benefit of that is: We get rid of a `!` and the logic is slightly more straightforward.